src/Controls/src/Core/Application.cs

```diff
@@ -51,7 +51,10 @@ internal Application(bool setCurrentApplication)
 
 			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<Application>>(() => new PlatformConfigurationRegistry<Application>(this));
 
-			_lastAppTheme = PlatformAppTheme;
+			_platformAppTheme = AppInfo.RequestedTheme;
+			_lastAppTheme = _platformAppTheme;
+
+			_platformLayoutDirection = AppInfo.RequestedLayoutDirection;
```

Comment:
Are these only called once on startup now? 👍

Reply:
It is called in 2 places.

This is in the ctor for the current value, but then when things change, the `void IApplication.ThemeChanged()` method updates this.

On iOS this is when the traits change, Android when the configuration changes and on Windows when the message loop gets a settings/theme change message.